README.md

```diff
@@ -151,6 +152,26 @@ Used for creating the default HTML needed to use a woonerf application.  Accepts
 * `staticHost`: (optional) The host server of the static files.  This gets prepended to an expected assets folder for the static files.  The files loaded will be: `${staticHost}assets/favicon.ico`, `${staticHost}assets/index.css` and `${staticHost}assets/index.js`.  If omitted, the host path will be an empty string.
 * `title`: The string to insert into the `title` tag.
 
+### message
+
+`message(defaultMessage, ?key, ...args)`
```

Comment:
In the implementation, `key` is not optional if this function is only used as a way to get to sprintf.  It may be confusing to someone trying to do:

```javascript
message('hello %s %s', 'bob', 'tim')
```

...because they'd get only 'tim' inserted.